conmon/conmon.c

```diff
@@ -632,6 +636,45 @@ int main(int argc, char *argv[])
 	cpid = atoi(contents);
 	ninfo("container PID: %d", cpid);
 
+	/* Setup endpoint for attach */
+	struct sockaddr_un attach_addr = {0};
+	_cleanup_close_ int afd = -1;
+	int asfd = -1;
+
+	if (!exec) {
+		attach_addr.sun_family = AF_UNIX;
+		snprintf(attach_sock_path, PATH_MAX, "/var/run/%s-attach", cid);
```

Comment:
I.e, make symlink `/tmp/foo`-> $bundledir, then bind to `/tmp/foo/attach-socket`, then delete /tmp/foo.
Connecting does the same.

This is nicer because then the socket is in the right place wrt cleanup, and things like that.